srcpkgs/sbt/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'sbt'
+pkgname=sbt
+version=1.2.1
+revision=1
+hostmakedepends="openjdk wget"
```

Comment:
Isn't openjdk unnecessary at `build time` since you effectively only unpack and move the files